src/egon/data/datasets/pypsaeursec/__init__.py

```diff
@@ -670,26 +670,53 @@ def lines_to_etrago(neighbor_lines=neighbor_lines, scn="eGon100RE"):
     lines_to_etrago(neighbor_lines=neighbor_lines, scn="eGon2035")
 
     # prepare and write neighboring crossborder links to etrago tables
-    def links_to_etrago(neighbor_links, scn="eGon100RE"):
+    def links_to_etrago(neighbor_links, scn="eGon100RE", extendable=True):
```

Comment:
This function has no documentation. Did you added this one or was it already there before?

Reply:
The function was already there (without documentation), I just add a parameter to it. But I can definitely document it.